### PR TITLE
cleanup(OWNERS): remove inactive approvers

### PR DESCRIPTION
**Pull request by jasondellaluce**
**What type of PR is this?**

/kind cleanup

/kind documentation

**Any specific area of the project related to this PR?**

/area build

**What this PR does / why we need it**:

For https://github.com/falcosecurity/evolution/issues/157, this PR moves inactive approvers (who had very little or zero contributions over the past 6 months) from `approvers` to `emeritus_approvers` in OWNERS.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by leodido — CHANGES_REQUESTED** [review]
https://github.com/falcosecurity/falco/issues/2115#issuecomment-1176048415

**Comment by FedeDP** [comment]
This PR was opened 2 weeks ago. It is blocked by the pending "change requests" (but the reviewer decided to step down). IMHO, the change looks legit because I have not seen any relevant activity from them in the last 6 months (given our governance).

I think all the remaining maintainers agree (cc @leogr @mstemm @gnosek  @Andreagit97 @Molter73 ). Thus, I'd ask @jasondellaluce to open a new PR to unlock this one.
Thank you!

**Comment by jasondellaluce** [comment]
> This PR was opened 2 weeks ago. It is blocked by the pending "change requests" (but the reviewer decided to step down). IMHO, the change looks legit because I have not seen any relevant activity from them in the last 6 months (given our governance).
> 
> I think all the remaining maintainers are on board (cc @leogr @mstemm @gnosek @Andreagit97 @Molter73 ). Thus, I'd ask @jasondellaluce to open a new PR to unlock this one. Thank you!

Closing this and opened https://github.com/falcosecurity/libs/pull/489.

/close

**Pull request closed without merging**
